File: problemas/iniciante/2779/2779.lua
n = tonumber(io.read(), 10)
m = tonumber(io.read(), 10)

figurinhas = {}
for i = 1, m do
  x = tonumber(io.read(), 10)
  figurinhas[x] = true
end

contador = 0
for i = 1, n do
  if figurinhas[i] then
    contador = contador + 1
  end
end

print(n - contador)

File: problemas/iniciante/2779/2779.r
input <- file('stdin', 'r')

n <- as.integer(readLines(input, n = 1))
m <- as.integer(readLines(input, n = 1))

figurinhas <- array(FALSE, dim = c(n + 1))
for (i in 1:m) {
  x <- as.integer(readLines(input, n = 1))
  figurinhas[x] <- TRUE
}

contador <- 0
for (i in 1:n) {
  if (figurinhas[i] == FALSE) {
    contador <- contador + 1
  }
}

write(paste(contador), '')